tests/FSharp.Compiler.ComponentTests/EmittedIL/MethodImplAttribute/MethodImplAttribute.Async.fs

```diff
@@ -0,0 +1,7 @@
+// Regression test for DevDiv:212424
+// "NoInlining attribute not emitted into IL"
+module M
+open System.Threading.Tasks
+open System.Runtime.CompilerServices
+[<System.Runtime.CompilerServices.MethodImplAttribute(System.Runtime.CompilerServices.MethodImplOptions.Async)>]
```

Comment:
I assume this will fail when targeting desktop framework, you might need to use `TheoryForNETCOREAPPAttribute` in the test declaration.